http.go

```diff
@@ -533,3 +554,28 @@ func isAllCaps(s string) bool {
 	}
 	return true
 }
+
+func readAgentsFile(path string) (*agent.Visitor, error) {
+	var v agent.Visitor
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	if err := json.NewDecoder(f).Decode(&v); err != nil {
+		return nil, err
+	}
+	return &v, nil
+}
+
+func defaultAgentFile() string {
```

Comment:
not in this PR but cen we add an AGENT_FILE environment variable to this please. I try to avoid filling up $HOME with tiny files.